docs/source/infrastructure.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: Infrastructure
+---
+
+# Infrastructure
+
+Pay uses Terraform to manage all infrastructure residing in AWS staging & production environments.
+
+All Terraform code is located in the `/terraform` directory in the [pay-omnibus repository][this-repo]:
+
+```sh
+/terraform
+  /account # Manages AWS account configuration
+  /modules # Shared modules used by staging & production
+  /staging # Manage staging infrastructure
+```
+
+## Getting started with Terraform
+
+Install Terraform `v0.12.18` or later. You can use [tfenv](https://github.com/tfutils/tfenv) to do this.
+ 
+## Terraform Plugins
+
+Pay's Terraform uses the AWS and Pass plugins.
+
+The Pass plugin is used to decrypt secrets located in te `/secrets` directory and must be manually installed.
+
+1. Download the [Pass plugin binary](https://github.com/camptocamp/terraform-provider-pass/releases)
+2. Move the downloaded binary into your Terraform plugins directory:
+
+```sh
+mv terraform-provider-pass ~/.terraform.d/plugins
+```
+
+## Run Terraform with AWS Credentials
+
+Ensure you have access to the Pay AWS account by assuming a role delegated via the gds-users account.
+
+Use [aws-vault](https://github.com/99designs/aws-vault) to manage credentials and execute commands using temporary credentials.
```

Comment:
Current RE recommendation is actually to use `gds-cli` for this (which uses aws-vault under the hood, so no real difference). Could link out to https://reliability-engineering.cloudapps.digital/iaas.html#use-the-command-line

I think recommending aws-vault is fine until we've set that up though.